### PR TITLE
Removes empty <div> or <p> elements from pages

### LINKED ISSUE #3090: Should empty elements be pruned from pages?
While working on #3085, I noticed that there were quite a bit more empty element occurrences in translated content. To be specific:

- 610 occurrences of empty div elements e.g. `<div></div>` in 448 files
- 66 occurrences of empty p elements e.g. `<p></p>` in 48 files

It seems that the Builder is generating extra empty elements and I can't see a good reason to keep these empty elements around. Especially if the empty elements are without a class/id selector, it leads me to believe they indeed are empty elements and will not be used elsewhere by JS/etc so its probably safe to remove them. That is, if it is outside of a code snippet and "ready" for removal.

### PR DESCRIPTION
**Pull request by tannerdolby**
Fixes #3090

### CODE REVIEW

**Review by sashasushko — COMMENTED** [review]
Hi @tannerdolby. I checked RU locale. There are no files from #3085. Is it ok?

**Comment by tannerdolby** [comment]
Hi @sashasushko. That should be ok. I will double check the RU locale files in #3085 just to be sure nothing was missed.

**Review by sashasushko — APPROVED** [review]
Approve from @mdn/yari-content-ru

**Review by nathipg — APPROVED** [review]
Approved from @mdn/yari-content-pt-br

**Review by irvin — APPROVED** [review]

**Comment by SphinxKnight** [comment]
The files for French (currently conflicting) have been converted into Markdown, I guess most of them can be removed from this PR now (there might be a handful of empty HTML elements in tables which were not converted).
@tannerdolby, let me know if I can help with that.

**Comment by tannerdolby** [comment]
@SphinxKnight Yeah, it seems quite a few French files are conflicting. If you have the time/capacity, feel free to propose the changes here and I'm happy to do the "quick" commit through GitHub UI, or I can get around to removing the conflicting French files later tonight :)

**Comment by SphinxKnight** [comment]
> @SphinxKnight Yeah, it seems quite a few French files are conflicting. If you have the time/capacity, feel free to propose the necessary changes here and I'm happy to do the "quick" commit through GitHub UI, or I can get around to removing the conflicting French files later tonight :)

Here it is, I've reset the files as upstream (sorry for the forced-push, forgot a bunch of files in my first commit attempt :x)

**Review by SphinxKnight — APPROVED** [review]

**Comment by tannerdolby** [comment]
Thank you for fixing those conflicts @SphinxKnight :)

**Review by pje1740 — APPROVED** [review]
Approved from @mdn/yari-content-ko 
Thank you! :)

**Review by mfuji09 — APPROVED** [review]
I have reviewed 18 files in ja locale.
A file is already deleted, so I have commited it.

**Comment by hochan222** [comment]
I think we only have @mdn/yari-content-fr left.

@irvin And there seems to be some conflict.

Have a nice day :)

**Comment by SphinxKnight** [comment]
I approved previously for @mdn/yari-content-fr :)

**Comment by Graywolf9** [comment]
Hi! Thank you so much! As `fr` in `es` locale we are updating to md, so by now all the files are outdated, but everyting seems to be ok

**Review by Graywolf9 — APPROVED** [review]

File: files/ko/web/api/console/index.html [hunk 1]
```diff
@@ -0,0 +1,296 @@
+---
```
Comment (irvin) [inline files/ko/web/api/console/index.html:1]: un-need file

**Review by irvin — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/html/element/var/index.html [hunk 2]
```diff
@@ -6,10 +6,22 @@
   - Web
 translation_of: Web/HTML/Element/var
 ---
+<<<<<<< HEAD
```
Comment (irvin) [inline files/zh-cn/web/html/element/var/index.html]: broken due to rebaase

**Review by irvin — CHANGES_REQUESTED** [review]

File: files/ko/web/api/console/index.html (hunk 1 above)
Comment (tannerdolby) [reply files/ko/web/api/console/index.html:1]: ok, will fix now

File: files/zh-cn/web/html/element/var/index.html (hunk 2 above)
Comment (tannerdolby) [reply files/zh-cn/web/html/element/var/index.html]: yeah, the merge conflict must have slipped through during the rebase. I will update shortly

**Review by JuanVqz — APPROVED** [review]

**Review by irvin — APPROVED** [review]

**Pull request merged**